aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move

```diff
@@ -61,5 +61,9 @@ module aptos_framework::aggregator_v2 {
 
     public native fun snapshot<Element>(aggregator: &Aggregator<Element>): AggregatorSnapshot<Element>;
 
+    public native fun create_snapshot<Element: copy + drop>(value: Element): AggregatorSnapshot<Element>;
```

Comment:
What's the potential use case of this? 
Do we want someone to be able to create a snapshot without an underlying aggregator? That would change the semantics and our underlying data structures as well.